master/internal/db/postgres_tasks.go

```diff
@@ -0,0 +1,59 @@
+package db
+
+import (
+	"github.com/o1egl/paseto"
+	"github.com/pkg/errors"
+
+	"github.com/determined-ai/determined/master/pkg/model"
+)
+
+// initTaskSessions creates a row in the task_sessions table.
+func (db *PgDB) initTaskSessions() error {
+	_, err := db.sql.Exec("DELETE FROM task_sessions")
```

Comment:
Since this is just run on startup, I think we're fine to use `TRUNCATE TABLE task_sessions` and it may be a bit quicker. https://www.postgresql.org/docs/9.1/sql-truncate.html